.github/workflows/build-cache.yml

```diff
@@ -26,14 +26,14 @@ jobs:
       run: rustup toolchain install nightly-2025-05-14 --component clippy,rustfmt
     - name: Install Rust toolchain
       run: rustup toolchain install nightly-2025-02-14 --component rust-src
-    - name: Install Rust toolchain 1.85 (stable)
-      run: rustup toolchain install 1.85
+    - name: Install Rust toolchain 1.88 (stable)
```

Comment:
Is this change needed?

Reply:
we already changed it to 1.88 in the runs a few weeks ago, so wanted to make this consistent